README.md

```diff
@@ -292,6 +292,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Run Lighthouse in CI using GitHub Actions](https://github.com/treosh/lighthouse-ci-action)
 - [Continuous Benchmarking and Benchmark Visualization for Go](https://github.com/bobheadxi/gobenchdata)
 - [Size Limit Action](https://github.com/andresz1/size-limit-action) - Comments cost comparison of your JS in PRs and rejects them if limit is exceeded.
+- [Check my bundlephobia Action](https://github.com/carlesnunez/check-my-bundlephobia) - Comments new and modified package size according to bundlephobia.io website. Let's you also reject PR on threshold surpassed.
```

Comment:
Please make this description more succinct - something along the lines of "Comments new and modified package size according to bundlephobia.io and allows rejection on threshold surpassed."